src/systems/systems/src/navigation/adirs.rs

```diff
@@ -1129,7 +1129,7 @@ impl InertialReference {
         // based on Theta and Phi.
         let g = Acceleration::new::<meter_per_second_squared>(9.81);
```

Comment:
You can check UpdateContext::acceleration_plane_reference_filtered_ms2_vector, I think it might do what you want already.

Reply:
Unfortunately, filtering the accelerations too much would affect the guidance and flight controls, and it seems like the filter is pretty strong. I'd need the unfiltered accelerations in g for this